examples/step-77/step-77.cc

```diff
@@ -633,8 +693,6 @@ namespace Step77
         output_results(refinement_cycle);
 
         computing_timer.print_summary();
-
-        std::cout << std::endl;
```

Comment:
You probably want to undo this change.

Reply:
I have removed all changes to step-77. I have backed the changes here: https://github.com/peterrum/dealii/tree/nox_tutorial